tests/h/auth/services_test.py

```diff
@@ -206,9 +210,308 @@ def test_it_provides_user_service(self, pyramid_request, user_service):
         assert svc.usersvc == user_service
 
 
+class TestOAuthServiceVerifyJWTBearer(object):
+    """ Tests for ``OAuthService.verify_jwt_bearer`` """
```

Comment:
There's a couple of prospector / linter warning about this docstring too (and the one below)